Content.Server.Database/Model.cs

```diff
@@ -117,7 +117,7 @@ public class Job
         [Column("priority")] public DbJobPriority Priority { get; set; }
     }
 
-    public enum DbJobPriority
+    public enum DbJobPriority : byte
```

Comment:
I'm not sure on changing the DB stuff, probably best to leave it as it might need a new migration.

Reply:
OK, returning to previous version.